transcendence-app/src/chat/chat.controller.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Post,
+  Query,
+  ServiceUnavailableException,
+  UnprocessableEntityException,
+  UseGuards,
+} from '@nestjs/common';
+import { ChatService } from './chat.service';
+import { AuthenticatedGuard } from '../shared/guards/authenticated.guard';
+import {
+  ApiBadRequestResponse,
+  ApiCreatedResponse,
+  ApiForbiddenResponse,
+  ApiOkResponse,
+  ApiServiceUnavailableResponse,
+  ApiTags,
+  ApiUnprocessableEntityResponse,
+} from '@nestjs/swagger';
+import { Chat } from './chat.domain';
+import { MAX_ENTRIES_PER_PAGE } from '../shared/constants';
+import { CreateChatDto } from './dto/create-chat.dto';
+import { User } from '../user/user.domain';
+import { User as GetUser } from '../user/decorators/user.decorator';
+import { ChatsPaginationQueryDto } from './dto/chat.pagination.dto';
+
+@Controller('chatroom')
```

Comment:
suggestion: To use the same controller for the chatrooms and 1:1 chats, as we agreed in the last meeting, we could add the `'chat'` prefix to this controller and the `'room'` prefix to the POST and GET routes for the chat rooms.

```
@Controller('chat')
...
@Post('room')
...
@Get('room')
```
The routes to create a chat becomes `POST /chat/room` and to get all the chats `GET /chat/room`

Reply:
Ok, I'll do it